app/helpers/Config/UploadsConfig.php

```diff
@@ -10,12 +10,22 @@ class UploadsConfig extends Nette\Object {
   protected $removalThreshold;
 
   /**
+   * @var int A limit for sizes of uploaded file previews (in bytes)
+   */
+  protected $maxPreviewSize;
+
+  /**
    * UploadsConfig constructor.
    * @param array $config
    * @internal param string $removalThreshold
    */
   public function __construct(array $config) {
     $this->removalThreshold = Arrays::get($config, "removalThreshold", "1 day");
+    $this->maxPreviewSize = Arrays::get($config, "maxPreviewSize", 65536);
```

Comment:
Should not we place this also into neon config file (at least as an example of what can be configured)?

Reply:
Probably. However, thanks to us copying the config.neon.example for every deployment, our configs are kinda crowded. Maybe we should just document this stuff elsewhere.

Reply:
OK